### PR TITLE
Add community documentation

### PR DESCRIPTION
**Pull request by Naylin15**
Hi, @ajsb85 

This PR adds the files requested in the issue #21 

Thank you

### CODE REVIEW

**Comment by Naylin15** [comment]
Hi, @ajsb85 

This PR is ready, I added the GLPI banner for the Readme and twitter badge

Thank you

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- File a GitHub issue only for bugs or feature requests related to the code **in this repository**. For other topics you can get more information in the README file. -->
+  
+### Observed Results:
```
Comment (ajsb85) [inline .github/ISSUE_TEMPLATE.md:3]: Remove the colon, it's a subtitle

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+<!-- This could be a description, error output, steps to reproduce, a feature missed, etc. -->
+  
+### Expected behavior:
```
Comment (ajsb85) [inline .github/ISSUE_TEMPLATE.md:7]: Remove the colon, it's a subtitle

File: README.md [hunk 3]
```diff
@@ -1,41 +1,62 @@
...
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
+Want to file a bug, contribute some code, or improve documentation? Excellent! Read up on our
+guidelines for [contributing](./CONTRIBUTING.md) and then check out one of our issues in the [Issues Dashboard](https://github.com/pluginsGLPI/telegrambot/issues).
 
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
```
Comment (ajsb85) [inline README.md]: GLPI registered brandy by Teclib'

File: README.md [hunk 4]
```diff
@@ -1,41 +1,62 @@
...
+## Copying
 
-    You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+* **Code**: you can redistribute it and/or modify
```
Comment (ajsb85) [inline README.md:60]: Missing GLPI registered brand by Teclib'

**Review by ajsb85 — APPROVED** [review]
With some minor changes request

**Pull request merged**